endpoints/openrtb2/auction.go

```diff
@@ -1653,3 +1654,23 @@ func checkIfAppRequest(request []byte) (bool, error) {
 	}
 	return false, nil
 }
+
+func getIntegrationTypeFromRequest(req *openrtb_ext.RequestWrapper) (string, error) {
+	reqExt, err := req.GetRequestExt()
+	if err != nil {
+		return "", err
+	}
+	reqPrebid := reqExt.GetPrebid()
+	return reqPrebid.IntegrationType, nil
+}
+
+func validateIntegrationType(req *openrtb_ext.RequestWrapper, integrationTypes map[string]bool) error {
```

Comment:
Are you planning to add test coverage for these functions in this PR or a future related PR?

Reply:
Yeah I'll add more test coverage in this PR!